src/main/java/ru/smirnovv/MyCollectors.java

```diff
@@ -4,7 +4,27 @@
 
 import static com.google.common.base.Preconditions.checkArgument;
 
-public class MyCollectors {
+/**
+ * This class contains methods, that returns new collectors.
+ */
+public final class MyCollectors {
+    /**
+     * Don't let anyone instantiate this class.
+     */
+    private MyCollectors() {
+    }
+
+    /**
+     * Return the new collector described by the MaxBoundedTreeSet methods.
```

Comment:
Returns a collector that accumulates the input elements into a bounded set of maximum ones.